qiskit/optimization/problems/quadratic_program.py

```diff
@@ -53,6 +54,16 @@ class QuadraticProgramStatus(Enum):
     INFEASIBLE = 1
 
 
+def _update_var_list(var_list: List[Variable], variable: Variable) -> List[Variable]:
```

Comment:
It would be better to let `update_var_list` and `update_var_dict` be `@staticmethod`.

Reply:
See my PR https://github.com/JosDenmark/qiskit-aqua/pull/1